shared/AppInsightsCore/Tests/Unit/src/ApplicationInsightsCore.Tests.ts

```diff
@@ -999,6 +998,30 @@ export class ApplicationInsightsCoreTests extends AITestClass {
             }
         });
 
+        // this.testCase({
```

Comment:
This is how I initially confirmed that this was the issue (after adding the test hooks).

Let me uncomment this to validate it again.